examples/Android/ScheduleApp/.gitignore

```diff
@@ -12,3 +12,5 @@
 /captures
 .externalNativeBuild
 .cxx
+app/release/
+app/debug/
```

Comment:
Is this app dir created in root directory? If yes, it would be better to add slash (/app/.). Do do we need anything from app directory (not sure where apk file is created). If no, suggest adding whole app directory (/app/ or app/).

Reply:
in app directory there are also sources and build files, we can not ignore them